Framework/DataHandling/src/LoadILLIndirect.cpp

```diff
@@ -291,14 +280,11 @@ void LoadILLIndirect::loadDataIntoTheWorkSpace(
   // Then add Simple Detector (SD)
   for (int i = 0; i < dataSD.dim0(); ++i) {
 
-    // just copy again the time binning axis to every spectra
-    m_localWorkspace->setSharedX((spec + nb_monitors + i),
-                                 m_localWorkspace->sharedX(0));
-
+	
     // Assign Y
     int *dataSD_p = &dataSD(i, 0, 0);
-    m_localWorkspace->mutableY(spec + nb_monitors + i)
-        .assign(dataSD_p, dataSD_p + m_numberOfChannels);
+	const HistogramData::Counts histoCounts(dataSD_p, dataSD_p + m_numberOfChannels);
+	m_localWorkspace->setHistogram((spec + nb_monitors + i), histoBinEdges, std::move(histoCounts));
```

Comment:
Careful, have a look at the original code again: There is no error in this case (for some reason). Your new code auto generates errors so you changed the behavior. To avoid that specify it directly by passing `CountSTandardDeviations(m_numberOfChannels, 0.0)` to `setHistogram` as last argument.